rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc

```diff
@@ -0,0 +1,52 @@
+= Activity Library Explorer 
+
+In the Activity Library Explorer you can view all the activity templates assigned to your organization. All users in your organization with the privilege *Activity Library Administration* have access to the same Activity Library.
+
+If another user adds an Activity, you see the change after refreshing the Activity Library Explorer.
+
+image::activity-library-explorer.png["The Activity Librar Explorer window."]
+
+== Toolbar 
```

Comment:
I realize that this is a "tour" of the UI, but we sort of want to move away from UI-centric docs and toward user-task docs. I'm wondering if it's necessary to document the toolbar?

Reply:
You could flip it around and have the user tasks first and point to the toolbar for how to accomplish each task.